iOS Client/Store/MBPodcastsStore.swift

```diff
@@ -57,3 +57,25 @@ class MBPodcastsStore {
         }
     }
 }
+
+public enum PodcastStream: String {
```

Comment:
this should live with the models, since it is referenced by the Podcast model. 

These models are the center of the Onion. If we define it here, at this outer layer, then it violates the Dependency Inversion Principle of Clean Code Architecture for other models to know about this one.

Again, I'm not going to fret over this.

Reply:
Interesting so where are you suggesting to put this?

Reply:
I'd say `Models` folder, maybe in its own file. 

On a side note, our DTOs should really not be in Models, but with the client code probably. Again, not a huge deal.